cli/command/utils.go

```diff
@@ -115,5 +117,23 @@ func PruneFilters(dockerCli Cli, pruneFilters filters.Args) filters.Args {
 		pruneFilters.Add(parts[0], parts[1])
 	}
 
+	// Server APIs older than 1.29 will just ignore label filters.
```

Comment:
@vdemeester @dnephin is right. This client-side fix fixes it for older daemons. I have a moby/moby PR that is almost ready, fixing it server side (returning error for unknown prune filters).
It is gonna be part of 17.06 as 17.05 is about to get released, and we still have to design and discuss a part of this server-side fix. I am having troubles with `docker system prune --filter ...` as the CLI, for `system prune`, calls `container prune`, `volume prune`, `image prune` and `network prune` passing the same filters to all of them... so my to-be PR is currently braking because of this.
Thanks for reading.